script.module.tvdbsimple/addon.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.tvdbsimple" name="tvdbsimple" version="1.0.6" provider-name="phate89">
+    <requires>
+        <import addon="xbmc.python" version="3.0.0"/>
+        <import addon="script.module.requests" version="2.9.1"/>
+    </requires>
+    <extension point="xbmc.python.module" library="lib" />
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en">tvdbsimple module</summary>
+        <description lang="en">Helper module for thetvdb site access</description>
+        <platform>all</platform>
+        <license>GPLv3</license>
```

Comment:
GPL-3.0-only or GPL-3.0-or-later depending on the license in use (see spdx.org)

Reply:
Got no clue which one is the correct one.

Reply:
Yeah, the author does not specify it. Leave it for now and maybe change it to `GPL-3.0-or-later` in a future update (it's the more generic one)